tests/test_app/TestApp/Model/Table/InvoicesTable.php

```diff
@@ -13,6 +13,7 @@ public function initialize(array $config)
             'contain' => [
                 'InvoiceItems.InvoiceItemProperties',
                 'InvoiceItems.InvoiceItemVariations',
+                'InvoiceTypes',
```

Comment:
This change doesn't make sense to me. Why would you use a belongs to association in the behavior config? A belongsTo associated record would be a parent and it shouldn't be duplicated. The duplicated record would have the same foreign key as original record and thus the same parent.